src/uct/api/v2/uct_v2.h

```diff
@@ -506,6 +549,25 @@ ucs_status_t uct_md_mkey_pack_v2(uct_md_h md, uct_mem_h memh,
                                  void *rkey_buffer);
 
 
+/**
+ * @ingroup UCT_MD
+ *
+ * @brief Locally attach to a remote memory.
+ *
+ * @param [in]  md            Handle to memory domain.
+ * @param [in]  mkey_buffer   Buffer with a packed memory handle as returned
+ *                            from @ref uct_md_mkey_pack_v2.
+ * @param [in]  params        Attach parameters, see @ref
+ *                            uct_md_mem_attach_params_t.
+ * @param [out] memh_p        Memory handle attached to a remote memory.
```

Comment:
is it really remote ?

Reply:
yes, it is memory allocated from an another CPU

Reply:
@dmitrygx then we probably should change `Buffer with a packed memory handle as returned` to `Buffer with a packed remote memory handle as returned`

Reply:
done